assets/vue/services/accessUrl.js

```diff
@@ -0,0 +1,45 @@
+export default {
+  async getUrl(id) {
+    const resp = await fetch(
+      `/main/admin/access_urls.php?action=fetch_url&url_id=${encodeURIComponent(id)}`,
```

Comment:
`fetch_url` doesn't exists in access_urls.php

You should use the `GET /api/access_urls/{id}` endpoint